src/BuildScriptGenerator/Node/NodeVersions.cs

```diff
@@ -4,8 +4,8 @@ namespace Microsoft.Oryx.BuildScriptGenerator.Node
 {
     public static class NodeVersions
     {
-        public const string NodeRuntimeBaseTag = "20191025.2";
-        public const string YarnCacheBaseTag = "20191025.2";
+        public const string NodeRuntimeBaseTag = "20191104.7";
```

Comment:
are we also updating the node runtimes with this? I would prefer not to change node and python runtime with this pr, you may have better reasoning though

Reply:
It is important to rely on the same base layers 'across' runtimes too..for example if Node base layer 'oryx-run-base' is old and PHP uses an updated(some new versions of packages were recently released) oryx-run-base, then we would not be sharing these common layers. Also since Nightly build has passed, I would not worry much about it..